src/main/java/com/google/sps/image/ImageSelection.java

```diff
@@ -27,6 +28,10 @@
   // Free to share and use commercially license
   private static final String USE_SHARE_FILTER = "&qft=+filterui:license-L2_L3_L4";
 
+  private static final String BACKUP_IMAGE1 = "https://cdn.pixabay.com/photo/2015/12/01/20/28/road-1072823__340.jpg";
+  private static final String BACKUP_IMAGE2 = "https://media.nationalgeographic.org/assets/photos/167/142/7dbe792c-eb3b-4743-9135-2e6087c7446c.jpg";
+  private static final String BACKUP_IMAGE3 = "https://www.wallpaperup.com/uploads/wallpapers/2017/01/18/1074860/b2d88573c9f3d799a4b23048e5dd9193-700.jpg";
```

Comment:
I think we should have a default image with some balloons/fireworks/confetti to be suitable for any occasion as well.

Reply:
Something like this maybe: https://previews.123rf.com/images/robisklp/robisklp1508/robisklp150800031/43577538-ballons-with-confetti.jpg

Reply:
Done.